### DISCUSSION TITLE
angled image

### DISCUSSION THREAD

**Discussion by vajonam** [blakeblackshear/frigate#6387] [Ask A Question] [answered]
I have a camera that is adjusted to capture as much as it can, however the road/horizon is tilted as a result.

is it possible to rotate just the bounding boxes to line up with the straighg lines or driveway? what is the suggestion here for mounting here ?

![image](https://user-images.githubusercontent.com/152501/236321988-cbf691bf-c9f0-422d-b853-efe6580d3ab3.png)

**Comment by NickM-27** [comment]
What do you mean by line up the bounding box? What you see is what was sent for detection, altering the bounding box would mean straying from what the model detected which is inaccurate

**Reply by vajonam** [reply]
so really can we rotate the image sent to the detector by a set amount, i know it is a wried ask. The more i think i think I really need to ensure the horizon is level for the zones etc to line up. also like a keystone correction for a projector so its easier to estimate where the bounding boxes are in relation to the zone.

**Reply by NickM-27** [reply, marked as answer]
You can do that yourself with ffmpeg output args
